### DISCUSSION TITLE
[Requests] A few usability improvements

### DISCUSSION THREAD

**Discussion by ddemtlapps** [agronick/aa-torque#29] [Ideas]
I found out about your Torque plugin yesterday on r/Mazda3 and dove right in! It's mighty promising and interesting!

Here are a few usability improvements I would suggest to make your app even better, in ascending order of usefulness:

Being able to export primary app settings as well as dashboards would help a lot, ideally in one shared file for both. The frequency of updates (which is otherwise nice!) and the need to uninstall every time means I seem to have to reconfigure this every time and although it's not that many options, it does make it cumbersome.

A way to visualize the results while we configure things in the phone app (with a Show Preview context menu item, for example) would be extra helpful to speed up the process. As much as I'd love to stay in the car while configuring, I just gotta be in the house with the family sometimes! 😉 I'm hoping you can fairly easily share the library that shows it in AA with the settings app on the phone, but I'm aware this could actually be a complex one!

With multiple dashboards, it would be a nice touch to use a UI element in the line above the title for example to show where the current dashboard is relative to the others. There are other ways to make this even cooler, but that would be a decent start and likely a quick win.

Thanks in advance for considering these among all others I am sure you have in mind! if only I wasn't so rusty, I would be tempted to contribute to the code...

**Comment by agronick** [comment]
Some good ideas here. I especially like the idea of showing the dots on the top to show what screen you are on. You don't need to uninstall before an upgrade even though Kingstaller tells you to. The  main settings will be exported eventually. They are just using the legacy settings API and need to be moved into the new way it does settings. It is something I have started working on but Android doesn't provide a good way to bridge the old style setetings and new style protocol buffers.

**Reply by ddemtlapps** [reply]
Interesting about the Kingstaller saying to uninstall first but it's not actually required... I definitely fell into that trap!

I declare a code 18 and blame being tired! 😉

**Reply by agronick** [reply]
So we got the preview and everything able to be backed up. It was a ton of work but the app is in a good position I feel. Now I can start looking for a real job.

**Comment by Pavloun** [comment]
Hello and thank you for your great app. I find it very useful and easy to operate :)

Is it possible to have displays and arrows for min/max values reset after a set time period or manually?
